modules/services/window-managers/fluxbox.nix

```
@@ -0,0 +1,135 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.xsession.windowManager.fluxbox;
+in {
+  meta.maintainers = [ maintainers.AndersonTorres ];
+
+  options = {
+    xsession.windowManager.fluxbox = {
+      enable = mkEnableOption "Fluxbox window manager";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.fluxbox;
+        defaultText = literalExpression "pkgs.fluxbox";
+        description = "Package to use for running Fluxbox WM.";
+      };
+
+      init = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Init configuration for Fluxbox, written to
+          <filename>~/.fluxbox/init<filename>. Look at the fluxbox(1) manpage
```

Missing a `/` 🙂

```suggestion
          <filename>~/.fluxbox/init</filename>. Look at the fluxbox(1) manpage
```

The same mistake also appears in the options below.

---

Also the man page reference can be made with `citerefentry`. That is, in total something like

``` xml
<filename>~/.fluxbox/init</filename>. See
<citerefentry>
  <refentrytitle>fluxbox</refentrytitle>
  <manvolnum>1</manvolnum>
</citerefentry>
for details.
```